mw-webapp/src/dataAccessLogic/renderCellValue/helpers/updateDayReport.ts

```diff
@@ -0,0 +1,24 @@
+import {DayReportDAL} from "src/dataAccessLogic/DayReportDAL";
+import {ColumnNameProps} from "src/dataAccessLogic/renderCellValue/renderCellItem";
+import {DayReport} from "src/model/businessModel/DayReport";
+
+/**
+ * Update DayReport
+ */
+export const updateDayReport = async (text: string, uuid: string, columnName: keyof ColumnNameProps, index: number) => {
```

Comment:
text -> updatedText
why do we need index if we are using uuid  ? maybe i missed up something

Reply:
1. Renamed
2. We need index in this case because we don't have uuid for collections StudentComments and LearnedForNextPeriod. It's a string arrays